### DISCUSSION TITLE
Usage 'Collection_List' in  Roles & Permissions

### DISCUSSION THREAD

**Discussion by KasperHeijma83** [directus/directus#12721] [Archive]
Can someone please inform what the usage of the field 'Collection_List' in Roles & Permissions exactly is and what kind of input is expected here? I can't seem to find it in any docs.

<img width="830" alt="image" src="https://user-images.githubusercontent.com/75990535/163141249-d694627a-b44c-4fce-91b8-f4306b524f38.png">

Many thanks in advance.

**Comment by rijkvanzanten** [comment]
Heya! That system field was removed quite a while ago as part of [this migration](https://github.com/directus/directus/blob/main/api/src/database/migrations/20211104A-remove-collections-listing.ts#L5). Did you run the database migrations when updating Directus?
